### PR TITLE
XIVY-13821 release 11.3 abstract

### PR DESCRIPTION
**Pull request by ivy-rew**
feel free to review on my machine: http://192.168.3.99/news
Selection_054

### CODE REVIEW

**Comment by ivy-rew** [comment]
I've already run the 'free' grammarly version on it :wink:

File: src/app/pages/news/11.3/meta.json [hunk 1]
```diff
@@ -2,6 +2,6 @@
   "logo": "/images/news/10.0/ivy-logo.svg",
   "title": "Axon Ivy 11.3",
   "slogan": "Variables Editor and Inscriptions",
-  "date": "Mai 15th, 2024",
+  "date": "Mai 22th, 2024",
```
Comment (weissreto) [inline src/app/pages/news/11.3/meta.json:5]: 23th

File: src/app/pages/news/11.3/abstract.html [hunk 2]
```diff
@@ -1,23 +1,34 @@
...
+      Run your workflow on the go - Axon Ivy has released a new version of the Axon Ivy Mobile App.
+      Our completely rewritten App is now available for Android and iOS alike.
     </p>
   </div>
   <div class="4u$ 12u$(xsmall)">
-    <h5>Other tweaks</h5>
+    <h5>Other features</h5>
     <ul>
-      <li>...</li>
+      <li>TLS Tester in Engine-Cockpit</li>
```
Comment (weissreto) [inline src/app/pages/news/11.3/abstract.html:21]: TLS Tester is not a big feature. Would put it to position 3

**Review by weissreto — APPROVED** [review]
Some small comments. Otherwise good

**Pull request merged**